007-design-docs.md

```diff
@@ -0,0 +1,94 @@
+# Design Documentation Proposal
+
+This document is intended to progress discussion about how to write and maintain [design documentation](https://en.wikipedia.org/wiki/Software_design_description) for Strimzi.
+
+The main goal of design documents for Strimzi is to make contributors more effective by encouraging all involved to think through the design and gather feedback from others. People often think the point of a design doc is to to teach others about some system or serve as documentation later on. While those can be beneficial side effects, they are not the goal in and of themselves.
```

Comment:
Is this still the main goal? It still seems at odds to the benefits listed

Reply:
How so? What would you say is the main goal?

Reply:
_The main goal of having design documents for Strimzi is so that behaviours of Strimzi as a whole and each of the components are well defined and are agreed upon._

Is what I am changing this to